### ISSUE TITLE
Cumulative XP

### ISSUE THREAD

**Issue by mattsigal** [Kamalisk/arkhamdb#102]
Something I've noticed is that the decklist tally of XP only shows the XP costs of cards currently in the deck.  However, since adding any card (even 0 XP ones) cost 1 XP during a campaign, this number doesn't reflect how much XP we've *actually* spent so far.

![screen shot 2017-06-12 at 13 32 29](https://user-images.githubusercontent.com/1421758/27046744-994b9f8e-4f73-11e7-9897-dacc355392f8.png)

It would be great if we also displayed the cumulative XP spent. This really would help as a sanity check - since most people have a record of how much XP they've gained each scenario, they could just quickly add them up and make sure they've spent the proper amount. 

E.g.:

![screen shot 2017-06-12 at 13 32 29](https://user-images.githubusercontent.com/1421758/27046837-ea6ae910-4f73-11e7-8bb4-3b446b3a1e71.png)

Even better would be if it was on a new line, and said something like "21 experience spent out of 23 earned" to give an even more thorough description.

**Comment by Kamalisk** [comment]
That value is for the deck as shown if built stand alone. 

The XP required depends entirely on the deck history, and so is shown in the deck upgrades section. It has no meaning when looking at a deck by itself. Any XP required from the perspective of a campaign is entirely dependent on how the deck compares to the previous version of the deck. 

Adding up the XP values would also not help as any kind of sanity check, as upgraded cards of the same name can give discounts on future versions, and 0 xp cards cost XP only if you happen to change the deck during the campaign (unless you exiled cards)

**Comment by mattsigal** [comment]
Yeah, I understand that difference.  But I mean - the information I'm requesting wouldn't be a simple sum, it would be a tally of total spent XP irregardless of source (accounting for adding 0XP cards, discounts from upgrading the same named card, etc) out of the total earned XP.  It could appear in the deck upgrades section, if that makes more sense.  

I think it would just be convenient to see - especially since most of these decks are played once and then sit for a month between releases, it is sometimes hard to recall how/if XP was spent (e.g., sometimes members of my playgroup decide that they really needed certain cards and spend their experience right after playing, other times they wait to see the new cards, et cetera).

**Comment by Kamalisk** [comment]
The text along side the deck is meant to be a summary of the deck itself, not any campaign that may be occurring. It should be possible to total the xp earned and put it down by the upgrade section. I planned to show this in the campaign tracker at least (if I ever finish it)

**Comment by mattsigal** [comment]
Yeah, I'm just trying to think of ways that will make campaign play a bit more straightforward. I find switching between campaigns a bit tricky.  

Ohhhhh, I didn't realize you were working on a campaign tracker - that would be so cool!
